src/ui/castlekeysmouse.pas

```diff
@@ -1300,19 +1304,41 @@ function TInputPressRelease.ToString: string;
     Result := Result + ', key repeated';
 end;
 
+function TInputPressRelease.KeyCharacter: char;
+begin
+  {$ifdef MSWINDOWS}
+  { It seems that GTK 1.3 for Windows cannot translate GDK_KEY_Escape and
+    GDK_KEY_Return to standard chars (#13 and #27). So I'm fixing it here. }
+  if Key = K_Escape then
+  Result := CharEscape else
+  if Key = K_Enter then
```

Comment:
Indent the lines with `Result` here, like

```
if Key = K_Escape then
  Result := CharEscape else
if Key = K_Enter then
  Result := CharEnter else
```

Better yet, place `else` on separate line:

```
if Key = K_Escape then
  Result := CharEscape
else
if Key = K_Enter then
  Result := CharEnter
else
```


See https://github.com/castle-engine/castle-engine/blob/master/doc/miscellaneous_notes/coding_conventions.txt .

Reply:
Oops. I've got it wrong somehow. I've thought it's done. I'll fix it right away.